### DISCUSSION TITLE
UVtools export to BMP (ESP32)

### DISCUSSION THREAD

**Discussion by Skywalkerf34** [sn4k3/UVtools#663] [General]
### Is your feature request related to a problem?

I am designing an  SLA 3D printer with ESP32 mcu
With this MCU it is possible to display BMP or PNG
do you think it might possible have this new function to export to those kind of files ?an 

### Describe the solution you'd like

Export to BMP or PNG would be great

**Comment by sn4k3** [comment]
UVtools can export to PNG and BMP, but ESP32 alone can't do a mSLA machine with current displays.
You will need another chip to handle images and more RAM than ESP32

**Comment by Skywalkerf34** [comment]
From my test I can do, there is 32MO chip now available  and  for files storage we can also use SDcard 
But can you explain How to export files in those format ?

**Reply by sn4k3** [reply]
On layer action -> Export to images

![image](https://user-images.githubusercontent.com/113281/220153076-ef705b04-c7a9-453c-90b9-6381ead8fcbc.png)

But if you slice with PrusaSlicer it always give you all PNGs on a zip where you can extract
